README.md

```diff
@@ -1,7 +1,135 @@
+between your organization name and the "Access Manager" link. This will take you
+to the Organization Settings screen. Copy your Organization ID and save it with
+your credentials.
+
+That's everything for Atlas. Now you're ready to move on to setting up Google Cloud!
+
+### Google Cloud
+
+If you don't already have a Google Cloud account, [sign up here](https://accounts.google.com/SignUp).
+We recommend using [Cloud Shell](https://cloud.google.com/shell) but if you prefer
+to work locally you can do that too. If you'd like to use Cloud Shell, just click
+this convenient button:
+
+[![Open in Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.svg)](https://shell.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/terraform-mean-cloudrun-mongodb)
+
+This will open a Cloud Shell window and clone a copy of this repository. Cloud
+Shell has your credentials built in, and Terraform is already installed, so
+you're ready to move on to running the demo. You can also [open Cloud Shell the normal way](https://cloud.google.com/shell/docs/using-cloud-shell)
+but you'll need to clone this repo yourself.
+
+If you want to work locally, you'll need to [install the `gcloud` tool](https://cloud.google.com/sdk/docs/install).
+Confidential to Mac users: `gcloud` is also available [via homebrew](https://formulae.brew.sh/cask/google-cloud-sdk).
+Once the install is done, run `gcloud init` to set up your environment. More
+information can be found [in the docs](https://cloud.google.com/sdk/docs/initializing).
+
+### Installing Terraform
+
+If you're using Cloud Shell, you've already got Terraform installed so feel free
+to move on to the next section. Otherwise, [download Terraform](https://www.terraform.io/downloads)
+and install it. A detailed walkthrough can be found [in this tutorial](https://learn.hashicorp.com/tutorials/terraform/install-cli).
+
+### Choosing a region
+
+Next, select a region for your infrastructure. For this demo it's okay to choose
+somewhere close to you, but for production use you may want to choose a different
+region. If you need some advice, check out [Best Practices for Compute Engine regions selection](https://cloud.google.com/solutions/best-practices-compute-engine-region-selection).
+
+For a list of available regions, refer Atlas's [Google Cloud provider documentation](https://www.mongodb.com/docs/atlas/reference/google-gcp/).
+Choose a region that's close to you and that supports the `M0` cluster tier. Make
+a note of both the Google Cloud region name and the Atlas region name, as you'll
+need both in the next step.
+
+### Configuring the demo
+
+If you haven't already, clone this repo. Run `terraform init` to make sure
+Terraform is working correctly and download the provider plugins.
+
+Then, create a file in the root of the repository called `terraform.tfvars` with
+the following contents, replacing placeholders as necessary:
+
+    atlas_pub_key        = "<your Atlas public key>"
```

Comment:
`(request "NOT_ORG_GROUP_CREATOR") The currently logged in user does not have the group creator role in organization 62757186a4a88a598211c228.`

Might want to mention `creator role` is required. In retrospect, this is really obvious, but I had just clicked through and accepted the defaults when creating API keys (what I saw seemed reasonable, I think it said "writer". :D

Reply:
the readme says to pick org owner [here](https://github.com/GoogleCloudPlatform/terraform-mean-cloudrun-mongodb/blob/mvp/README.md?plain=1#L36-L39) but maybe we should rework it to be clearer? I was thinking it would be nice to have screenshots of all these steps so it's easier to find things and make sure you're on track.

Reply:
😖 Doh, user error! Yup, my bad, thanks.